packages/ai-bot/main.ts

```diff
@@ -273,24 +285,14 @@ Common issues are:
             initialMessage.event_id,
           );
         }
-        let messageObject = {
-          body: 'some response, a patch',
-          msgtype: 'org.boxel.command',
-          formatted_body: 'some response, a patch',
-          format: 'org.matrix.custom.html',
-          command: {
-            type: 'patch',
-            id: getLastUploadedCardID(history),
-            patch: {
-              attributes: attributes,
-            },
+        let command = {
+          type: 'patch',
+          id: getLastUploadedCardID(history),
+          patch: {
+            attributes: attributes,
           },
         };
-        return await client.sendEvent(
-          room.roomId,
-          'm.room.message',
-          messageObject,
-        );
+        return await sendOption(client, room, command);
```

Comment:
is content and command the same? There seem to be 2 usess of sendOption
- sendOption(...,command)
- sendOption(...,content)

One is expecting a string and the other is expecting an object

Reply:
Naming isn't great, but that second one is also an object. The whole messageType stuff and custom parsing goes away with the function calling change https://github.com/cardstack/boxel/pull/933